test/audience.robot

```diff
@@ -27,17 +30,42 @@ Token with correct audience is accepted
     Should Contain   ${out}   404
 
 Token with correct audience in multiple option is accepted
+    [Tags]   audience
     ${uuid}   Generate UUID
     ${se_config}   Get SE config
     ${token}   Get token   scope=-s openid   opts=--aud="${se_config.endpoint} ${uuid}"
     ${url}   SE URL   audience-test-${uuid}
     ${rc}   ${out}   Curl Error   ${url}
     Should Contain   ${out}   404
 
-Token with invalid multiple audiences is rejected
+Token with invalid multiple audiences is rejected (permissive)
+    [Tags]   audience   permissive
+    ${out}   Get token with invalid multiple audience
+    ${ret}   Should Match Regexp   ${out}   40[134]
+
+Token with invalid multiple audiences is rejected (strict)
+    [Tags]   audience   strict
+    ${out}   Get token with invalid multiple audience
+    ${ret}   Should Match Regexp   ${out}   401
+
+
+*** Keywords ***
+
+Get token with random audience
```

Comment:
We should find a better name for the keyword, e.g. `Try get resource with token with random audience`

Reply:
Unfortunately I don't like too much this keyword name because of repetition of "with" and because it's quite long -- and the next one will be even longer if we apply the same modification. But I really cannot find one better, thus it's fine with me.